YIF_XUnitTests/Unit/YIF_Backend/Controllers/InstitutionOfEducationModeratorControllerTests.cs

```diff
@@ -62,5 +62,18 @@ public async Task UpdateSpecialtyDescription_EndpointReturnsOk()
             //Assert
             Assert.IsType<OkObjectResult>(result);
         }
+
+        [Fact]
+        public async void DeleteSpecialtyFromInstitutionOfEducation_ShouldReturnNoContent_IfEverythingIsOk()
+        {
+            // Arrange
+            _ioEModeratorService.Setup(x => x.DeleteSpecialtyToIoe(It.IsAny<SpecialtyToInstitutionOfEducationPostApiModel>()));
+
+            // Act
+            var result = await _testControl.DeleteSpecialtyFromIoE(new SpecialtyToInstitutionOfEducationPostApiModel());
+
+            // Assert  
+            Assert.IsType<NoContentResult>(result);
+        }
     }
```

Comment:
How about integration test for IoEAdminController?)

Reply:
Already fixed them in task #141.